docs/user/commandline_tools.rst

```diff
@@ -22,45 +22,49 @@ To see which SFT and Converter confs are installed try the ``geomesa env`` comma
 Running the command line tools
 ------------------------------
 
+.. note::
+
+    Some command names have changed in GeoMesa 1.3.0 see :doc:`commandline_old_commands`
+
 Run ``geomesa`` without any arguments to produce the following usage text::
 
     $ geomesa
     Usage: geomesa [command] [command options]
       Commands:
-        add-attribute-index    Run a Hadoop map reduce job to add an index for attributes
+        add-attributeindex     Run a Hadoop map reduce job to add an index for attributes
```

Comment:
hmmm actually i see that it used to be add-attribute-index but changed to just one dash?